CONTRIBUTING.md

```diff
@@ -47,39 +47,51 @@ We follow the
 [Fork and pull model](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/getting-started/about-collaborative-development-models#fork-and-pull-model)
 in GitHub.
 
-You need to first fork this repository, and you can later on open a pull request
+You need to first fork this repository, and you can later open a pull request
 to propose changes from your own fork to the **master** branch in this source
 repository.
 
 GitHub provides detailed instructions in
-[Fork a repo](https://docs.github.com/en/get-started/quickstart/fork-a-repo). In
-summary, you perform the follow steps to get your fork ready:
+[Fork a repo](https://docs.github.com/en/get-started/quickstart/fork-a-repo).
 
-1.  Set up Git and authentication with GitHub.com.
+If you're doing this for the first time, the easiest way to get going is with the `gh` CLI tool:
 
-    https://docs.github.com/en/get-started/quickstart/set-up-git
+1. Install the `gh` CLI tool following the [instructions](https://github.com/cli/cli#installation)
 
-2.  Fork the `k8s-config-connector` repo. Instructions below assumes you also
-    name your fork as `k8s-config-connector`. If you use a different name for
-    the fork, you should replace the commands with the right name.
+1. Clone the repo.  We recommend you to create the local clone under the path
+    `~/go/src/github.com/GoogleCloudPlatform`. 
 
-    https://docs.github.com/en/get-started/quickstart/fork-a-repo#forking-a-repository
+    ```shell
+    gh repo clone github.com/GoogleCloudPlatform/k8s-config-connector ~/go/src/github.com/GoogleCloudPlatform/k8s-config-connector
+    cd ~/go/src/github.com/GoogleCloudPlatform/k8s-config-connector
+    ```
 
-3.  Clone your forked repo to your dev machine.
+1. Create a fork of the repo in your github account: `gh repo fork`
 
-    https://docs.github.com/en/get-started/quickstart/fork-a-repo#cloning-your-forked-repository
+1. You can now inspect the remotes with `git remote -v`, you should see a remote
+   named `upstream` that is the main project, and a remote named `origin` that is your fork.
+   You will typically work locally, push branches to your fork (`origin`), and then send
+   Pull-Requests (PRs) to the main project (`upstream`) when they are ready.
 
-    We recommend you to create the local clone under the path
-    `~/go/src/github.com/YOUR_USERNAME`. This will help to avoid a few known
-    build frictions related to generated code.
 
-    ```shell
-    mkdir -p ~/go/src/github.com/YOUR_USERNAME
-    cd ~/go/src/github.com/YOUR_USERNAME
-    git clone https://github.com/YOUR_USERNAME/k8s-config-connector   # If you use ssh key auth, this will be git@github.com:YOUR_USERNAME/k8s-config-connector.git
-    ```
+### Run tests against a mock environment
```

Comment:
Shall we move this right after the title "Validate your environment"?

https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/860/files#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055R181

The instructions assumes Golang is already installed, which may not be the case.

Reply:
I'd rather we gave positive feedback as early as possible.  We're demonstrating that it's just a go program.  I added "install golang" as a pre-requisite.